tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -82,7 +82,8 @@ public class Program
             { "xunit.runner.visualstudio", "2.4.1" },
             { "Xunit.SkippableFact", "1.3.12" },
             { "Moq", "4.12.0" },
-            { "System.Linq.Async", "4.0.0" }
+            { "System.Linq.Async", "4.0.0" },
+            { ReferenceAssembliesPackage, ReferenceAssembliesVersion }
```

Comment:
Am I missing something... `ReferenceAssembliesPackage` and `ReferenceAssembliesVersion` don't seem to be defined?

Reply:
They're constants that were already defined elsewhere.